test/test_table.py

```diff
@@ -7,7 +7,8 @@
 from pykusto.expressions import column_generator as col, StringColumn, NumberColumn, AnyTypeColumn, BooleanColumn
 from pykusto.query import Query
 from pykusto.type_utils import KustoType
-from test.test_base import TestBase, mock_columns_response, mock_tables_response, mock_databases_response, MockKustoClient
+from test.test_base import TestBase, mock_columns_response, mock_tables_response, mock_databases_response, \
+    MockKustoClient, RecordedQuery, mock_getschema_response
 
 
 class TestTable(TestBase):
```

Comment:
wouldn't it be better to split this into a few files? maybe test auto complete or test_db, test_column and test_client (I tend towards the latter)

Reply:
Split to `test_client` and `test_client_fetch`